.drone.yml

```diff
@@ -131,7 +131,8 @@ trigger:
       - refs/tags/**
 
 steps:
-  - name: init
+  - &e2e-test-1-18
```

Comment:
Are you sure this syntax will work?

Reply:
woops that commit was not supposed to be there. Recent rebasing issue. fixed